src/reactviews/common/vscodeWebviewProvider.tsx

```diff
@@ -158,7 +162,10 @@ export function VscodeWebviewProvider<State, Reducers>({
                 }}
                 theme={theme}
             >
-                {children}
+                {
+                    // don't render webview unless necessary dependencies are initialized
+                    isInitialized() && children
```

Comment:
I made this change after chatting with @aasimkhan30.  It results in all webviews having defined state (with the default value set by each controller) instead of having to sprinkle undefined checks in each component that uses context.